src/Altinn.App.Api/Controllers/PartiesController.cs

```diff
@@ -21,6 +22,7 @@ namespace Altinn.App.Api.Controllers;
 /// </summary>
 [Authorize]
 [ApiController]
+[JsonSettingsName("EnumAsNumber")]
```

Comment:
Maybe the setting name should just be `AltinnApis` or something? Denoting that whatever settings are injected are what is required to make our own APIs stable?

Now maybe we should consider API conventions as well so that we don't have to remember this attribute everywhere?